jerry-core/ecma/builtin-objects/ecma-builtin-json.cpp

```diff
@@ -126,9 +105,12 @@ ecma_builtin_json_check_id (lit_utf8_byte_t *string_p, /**< start position */
   {
     string_p++;
     id_p++;
-    if (*id_p == '\0')
+    if (*id_p == LIT_CHAR_NULL)
     {
-      return !isalpha (*string_p) && !isdigit (*string_p) && *string_p != '$' && *string_p != '_';
+      return (!lit_char_is_ascii_alpha (*string_p)
```

Comment:
I said above no paragraph, just convenience :)

According to the spec, the lexer part of JSON accepts things like truetrue  and parses them as two token (which is not how identifiers usually handled). But two JSONValue-s cannot follow each other, so this will eventually aborts. But I think it is nicer to abort after the first ID.